src/main/java/com/faforever/api/player/PlayerService.java

```diff
@@ -29,6 +32,6 @@ public Player getPlayer(Authentication authentication) {
 
   public Player getById(Integer playerId) {
     return playerRepository.findById(playerId)
-      .orElseThrow(() -> new ApiException(new Error(ErrorCode.ENTITY_NOT_FOUND, playerId)));
+      .orElseThrow(() -> new ApiException(new Error(ErrorCode.PLAYER_NOT_FOUND, playerId)));
```

Comment:
.of ?

Reply:
Didn't check that class as it was auto-refactored ^^